server/models/Transaction.js

```diff
@@ -812,6 +812,15 @@ function defineModel() {
       return;
     }
 
+    // Skip if missing or misconfigured
+    const hostFeeShareCollective = await models.Collective.findByPk(HOST_FEE_SHARE_TRANSACTION_PROPERTIES.CollectiveId);
+    const hostFeeShareHostCollective = await models.Collective.findByPk(
+      HOST_FEE_SHARE_TRANSACTION_PROPERTIES.HostCollectiveId,
+    );
+    if (!hostFeeShareCollective || !hostFeeShareHostCollective) {
+      return;
+    }
```

Comment:
Oh, so it was an edge-case related to CollectiveId.
I think this makes perfect sense, we should later create a new DB seed export to make sure our environments are consistent.